cognite/client/_api/simulators/simulator_models.py

```diff
@@ -0,0 +1,213 @@
+            resource_cls=SimulatorModelRevision,
+            list_cls=SimulatorModelRevisionList,
+            filter=filter.dump()
+            if isinstance(filter, SimulatorModelRevisionsFilter)
+            else filter
+            if isinstance(filter, dict)
+            else None,
+        )
+
+    def retrieve(self, id: int | None = None, external_id: str | None = None) -> SimulatorModelRevision | None:
+        """`Retrieve simulator model revision <https://developer.cognite.com/api#tag/Simulator-Models/operation/retrieve_simulator_model_revisions_simulators_models_revisions_byids_post>`_
+
+        Retrieve multiple simulator model revisions by IDs or external IDs
+
+        Args:
+            id (int | None): The id of the simulator model revision.
+            external_id (str | None): The external id of the simulator model revision.
+
+        Returns:
+            SimulatorModelRevision | None: Requested simulator model revision
+
+        Examples:
+
+            Get simulator model revision by id:
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.simulators.models.revisions.retrieve(id=123)
+
+            Get simulator model revision by external id:
+                >>> res = client.simulators.models.revisions.retrieve(external_id="abcdef")
+
+        """
+        identifiers = IdentifierSequence.load(ids=id, external_ids=external_id).as_singleton()
+        return self._retrieve_multiple(
+            list_cls=SimulatorModelRevisionList,
+            resource_cls=SimulatorModelRevision,
+            identifiers=identifiers,
+            resource_path="/simulators/models/revisions",
+        )
+
+    def retrieve_multiple(
+        self,
+        ids: Sequence[int] | None = None,
+        external_ids: SequenceNotStr[str] | None = None,
+        ignore_unknown_ids: bool = False,
+    ) -> SimulatorModelRevisionList:
+        """`Retrieve simulator model revisions <https://developer.cognite.com/api#tag/Simulator-Models/operation/retrieve_simulator_model_revisions_simulators_models_revisions_byids_post>`_
+
+        Args:
+            ids (Sequence[int] | None): IDs
+            external_ids (SequenceNotStr[str] | None): External IDs
+            ignore_unknown_ids (bool): Ignore IDs and external IDs that are not found rather than throw an exception.
+
+        Returns:
+            SimulatorModelRevisionList: Requested simulator model revisions
+
+        Examples:
+
+            Get simulator model revisions by ids:
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.simulators.models.revisions.retrieve_multiple(ids=[1, 2, 3])
+
+            Get simulator model revisions by external ids:
+                >>> res = client.simulators.models.revisions.retrieve_multiple(external_ids=["abc", "def"])
```

Comment:
so there is an example of "retrieve_multiple", but there is no test, does it work?